MPower/app/src/main/res/drawable/ic_help_outline.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
These are Material Icons that I loaded as VectorDrawables.  Right-click on the drawable folder, then choose New -> Vector Asset.  The select the material icon.  You can also load your own svg's too.  In a few cases, I converted one of the ios images to svg and loaded it as a VectorDrawable.